Jobs.Fetcher.YouTube/YoutubeFetcherJobs.cs

```diff
@@ -77,8 +77,6 @@ public class ReprocessDailyVideoMetricsQuery : YoutubeFetcher {
                 if (item.Lifetime > 0) {
                     var ratio = Math.Abs((double) item.Lifetime - item.Total) / ((double) item.Lifetime);
                     if (forceFetch || Math.Abs(ratio) > comparisonThreshold && item.Lifetime > comparisonMinLimit) {
-                        Console.WriteLine("Reprocessing video {0}: daily views {1} from {4} to {5}, lifetime views {2} at {6} and ratio {3}"
-                                          , item.Id.VideoId, item.Total, item.Lifetime, ratio, item.DailyStart, item.DailyEnd, item.LifetimeDate);
                         Logger.Information("Reprocessing video {0}: daily views {1} from {4} to {5}, lifetime views {2} at {6} and ratio {3}"
                                            , item.Id.VideoId, item.Total, item.Lifetime, ratio, item.DailyStart, item.DailyEnd, item.LifetimeDate);
```

Comment:
```suggestion
                                           , item.Id.VideoId, item.Total, item.DailyStart, item.DailyEnd, item.LifetimeDate, item.Lifetime, ratio);
```

Reply:
I think I still prefer the format `$"Reprocessing video {item.Id.VideoId}" ... `, but maybe there is some limitation I'm not aware of

Reply:
I think all this information is not really that necessary so I just left the information of which video had its metrics reprocessed